### ISSUE TITLE
cannot compile matconvnet-1.0-beta25

### ISSUE THREAD

**Issue by lgen** [albanie/mcnSSD#13]
When i compile matconvnet-1.0-beta25, i get the fellowing error:

警告: Version 7.0 of the CUDA toolkit could not be found. If installed, set MW_NVCC_PATH
environment variable to location of nvcc compiler.
 
> In mexcuda (line 157)
  In vl_compilenn>mexcuda_compile (line 603)
  In vl_compilenn (line 489) 
错误使用 mex
未找到支持的编译器或 SDK。有关选项，请访问
http://www.mathworks.com/support/compilers/R2015b/glnxa64.html。

出错 mexcuda (line 157)
    [varargout{1:nargout}] = mex(mexArguments{:});

出错 vl_compilenn>mexcuda_compile (line 603)
mexcuda(args{:}) ;

出错 vl_compilenn (line 489)
      mexcuda_compile(opts, srcs{i}, objfile, flags) ;

My system setting is ubuntu16.04,cuda8.0 ,matlab2015b. And i can compile matconvnet-1.0-beta24 and matconvnet-1.0-beta23 successfully. Could you give me some help about solving this problem ? Thank you.

**Comment by albanie** [comment]
Ah, sorry to hear that.  Are you explicitly passing the location of the cuda toolkit when you compile matconvnet?  E.g. `vl_compilenn(...., 'cudaRoot', 'path/to/your/cuda-8', ...)` ?

**Comment by lgen** [comment]
Yes, i have tried, but it still doesn't work. Should i use a higher version of MATLAB ?

**Comment by albanie** [comment]
I'm not 100% certain to be honest, but it is worth a try if that is an option. Out of the box, I think 2015b is only guaranteed to be compatible with cuda 7.
